toonz/sources/tnztools/brushtool.cpp

```diff
@@ -1335,7 +1335,7 @@ void BrushTool::leftButtonDown(const TPointD &pos, const TMouseEvent &e) {
 
     /*--- ストロークの最初にMaxサイズの円が描かれてしまう不具合を防止する ---*/
     if (m_pressure.getValue() && e.m_pressure == 1.0)
-      thickness     = m_rasThickness.getValue().first;
+      thickness     = m_thickness.getValue().first;
```

Comment:
@manongjohn Thank you for tackling with this problem.
I myself inserted this line when I was in Ghibli, and it's obviously wrong. I couldn't realize it until today as Ghibli had mostly used Toonz Raster levels. 

Just one point I would like you to fix - it seems that this line seem to be 
`thickness     = m_thickness.getValue().first * 0.5;`
in order to get smooth stroke, since `computeThickness()` returns half of the thickness value.